contracts/src/s-token/STokensManager.sol

```diff
@@ -23,6 +24,7 @@ contract STokensManager is
 	mapping(address => EnumerableSet.UintSet) private tokenIdsMapOfOwner;
 	mapping(uint256 => string) private tokenUriImage;
 	mapping(uint256 => bool) public override isFreezed;
+	mapping(address => address) private imageDescriptorOfProperty;
```

Comment:
`descriptorOf` is more appropriate than `imageDescriptorOfProperty`. That's because it's not essentially limited to the image.

Also, how about making this a pablic and exposing a function called `descriptorOf(PROPERTY)`?

Reply:
I fix it -> descriptorOf(PROPERTY)